src/pmi/src/pmi_wire.c

```diff
@@ -606,18 +604,10 @@ int PMIU_cmd_output_v1(struct PMIU_cmd *pmicmd, char **buf_out, int *buflen_out)
 
     *buf_out = pmicmd->tmp_buf;
     *buflen_out = buflen;
-
-  fn_exit:
-    return pmi_errno;
-
-  fn_fail:
-    goto fn_exit;
```

Comment:
Why don't we just remove the labels? The function may fail, for example, out of memory or exceeding temp buffer limit or `pmicmd` fails validation. Even though we do not check these failures now, having the error return makes future modification easier.